### PR TITLE
Project rubric

### LINKED ISSUE #55: Get Project Rubric Endpoint
`GET /api/v1/instructors/:instructor_id/instructor_students/:student_id/project_templates?mod=1&project_number=2`
response must have 
- [x] student_id
- [x] project template id
- [x] rubric_category_template_id
- [x] rubric_category_name

### LINKED ISSUE #67: Get Project Rubric Sad Path/Edge Case

### PR DESCRIPTION
**Pull request by jequeza**
### Add
* ProjectTemplates Controller for the project rubric category request. Tests for Poros. 

### Why?
* Complete user story

Close #55

### CODE REVIEW

File: app/models/project_template.rb [hunk 1]
```diff
@@ -5,4 +5,11 @@ class ProjectTemplate < ApplicationRecord
   validates :name, :mod, :program, :project_number, presence: true
 
   enum project_type: [:solo, :paired, :team]
+
+
+  def self.get_project(program, mod, project_number)
```
Comment (b-enji-cmd) [inline app/models/project_template.rb:10]: Awesome query!

File: app/poros/project_rubric.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+              :project_number,
+              :rubric_template
+
+  def initialize(params)
+    @id = project(params)[0].id
+    @student_id = student(params).user.id
+    @project_template_id = project(params)[0].id
+    @mod = student(params).current_mod
+    @project_number = project(params)[0].project_number
+    @rubric_template = print_rubric(RubricTemplate.find(project(params)[0].rubric_template_id).rubric_template_categories)
```
Comment (b-enji-cmd) [inline app/poros/project_rubric.rb:15]: This is so clean!

**Review by b-enji-cmd — APPROVED** [review]
LGTM! You're killin it!
